packages/core/src/components/private/BaseCheckBox.tsx

```diff
@@ -13,11 +14,14 @@ export type Props = InputProps & {
   hideLabel?: boolean;
   /** Callback fired when the value changes. */
   onChange: (checked: boolean, value: string, event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Mark the checkbox as greyed out with a dash to indicate a neutral or inindeterminate state. */
+  neutral?: boolean;
 };
 
 class BaseCheckBox extends React.Component<Props & WithStylesProps> {
   static defaultProps = {
     checked: false,
+    neutral: false,
   };
 
```

Comment:
Going to need to add an airbnb prop type so that checked/neutral cannot be used at the same time.